modules/ROOT/pages/database-administration/index.adoc

```diff
@@ -1,114 +1,80 @@
-:description: Introduction to managing multiple active databases with Neo4j.
-[[manage-databases-introduction]]
-= Introduction
+[database-administration]]
+= Database administration
 
-[[manage-databases-concepts]]
-== Concepts
-
-With Neo4j {neo4j-version} you can create and use more than one active database at the same time.
-
-DBMS::
-Neo4j is a Database Management System, or _DBMS_, capable of managing multiple databases.
+Neo4j is a Database Management System, or DBMS, capable of managing multiple databases.
 The DBMS can manage a standalone server, or a group of servers in a cluster.
 
-Instance::
-A Neo4j instance is a Java process that is running the Neo4j server code.
-
-Transaction domain::
-A transaction domain is a collection of graphs that can be updated within the context of a single transaction.
-
-Execution context::
-An execution context is a runtime environment for the execution of a request.
-In practical terms, a request may be a query, a transaction, or an internal function or procedure.
-
-Graph::
-A data model within a database.
-
-Database::
 A database is an administrative partition of a DBMS.
 In practical terms, it is a physical structure of files organized within a directory or folder, that has the same name of the database.
-+
-In Neo4j {neo4j-version} each standard database contains a single graph.
+This chapter describes how to manage local and remote standard databases, composite databases, and database aliases.
+
+== Standard databases
+
+In Neo4j {neo4j-version}, each standard database contains a single graph.
 Many administrative commands refer to a specific graph by using the database name.
-+
-A database defines a _transaction domain_ and an _execution context_.
+
+A database defines a _transaction domain_ (a collection of graphs that can be updated within the context of a single transaction) and an _execution context_(a runtime environment for the execution of a request).
```

Comment:
you've added a brief description of these two when removing the concept section but not for `graph`, should it also have a description somewhere here like dbms and database still does?

Reply:
Yes - good catch 👍